lua/telescope/actions/init.lua

```diff
@@ -137,6 +137,13 @@ function actions.remove_selection(prompt_bufnr)
   current_picker:remove_selection(current_picker:get_selection_row())
 end
 
+-- Set working directory to the current selection's dir
+-- For file_browser this corresponds directly to the selection
+-- For find_files this sets it to the directory containing current selection
+function actions.set_cwd_to_selection(path)
```

Comment:
no matter what the other people said, this is not an action, this is a mere util function.

A action should ALWAYS be mapable to a picker, so it needs to get the signature `(prompt_bufnr)`, like all other functions in this file.

Reply:
I will just remove it from actions and place that inside individual functions. It is a single line anyway. Would that work?

Reply:
works for me